packages/opentelemetry-instrumentation-llamaindex/opentelemetry/instrumentation/llamaindex/base_tool_instrumentor.py

```diff
@@ -0,0 +1,62 @@
+from importlib.metadata import version as package_version, PackageNotFoundError
```

Comment:
:upside_down_face: (NIT) There is a lot of duplication here. Maybe it could be refactored sometime.

Reply:
Yeah I know :/ I need to refactor this